components/raftstore-v2/src/raft/peer.rs

```diff
@@ -619,6 +619,16 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             if let Some(instant) = self.peer_heartbeats.get(&p.get_id()) {
                 let elapsed = now.saturating_duration_since(*instant);
                 if elapsed >= ctx.cfg.max_peer_down_duration.0 {
+                    warn!(
```

Comment:
It will print every 60s, will it be noise?

Reply:
Why is there a down peer every 60s under normal circumstances??